catalog/plugin/catalog-plugin-metacardbackup/src/test/java/org/codice/ddf/catalog/plugin/metacard/backup/MetacardBackupPluginTest.java

```diff
@@ -127,54 +134,45 @@ public void testMetacardTransformerId() {
 
     @Test
     public void testRefresh() {
-        String newBackupDir = "target/temp";
         String newMetacardTransformer = "tika";
         Map<String, Object> properties = new HashMap<>();
-        properties.put("outputDirectory", newBackupDir);
         properties.put("metacardTransformerId", newMetacardTransformer);
         properties.put("keepDeletedMetacards", Boolean.TRUE);
         metacardBackupPlugin.refresh(properties);
         assertThat(metacardBackupPlugin.getKeepDeletedMetacards(), is(true));
-        assertThat(metacardBackupPlugin.getOutputDirectory(), is(newBackupDir));
         assertThat(metacardBackupPlugin.getMetacardTransformerId(), is(newMetacardTransformer));
     }
 
     @Test
     public void testRefreshBadValues() {
         Map<String, Object> properties = new HashMap<>();
-        properties.put("outputDirectory", 2);
         properties.put("metacardTransformerId", 2);
         properties.put("keepDeletedMetacards", "bad");
         metacardBackupPlugin.refresh(properties);
         assertThat(metacardBackupPlugin.getKeepDeletedMetacards(), is(false));
-        assertThat(metacardBackupPlugin.getOutputDirectory(), is(OUTPUT_DIRECTORY));
         assertThat(metacardBackupPlugin.getMetacardTransformerId(), is(METACARD_TRANSFORMER_ID));
     }
 
     @Test
     public void testRefreshEmptyStrings() {
         Map<String, Object> properties = new HashMap<>();
-        properties.put("outputDirectory", "");
         properties.put("metacardTransformerId", "");
         metacardBackupPlugin.refresh(properties);
-        assertThat(metacardBackupPlugin.getOutputDirectory(), is(OUTPUT_DIRECTORY));
         assertThat(metacardBackupPlugin.getMetacardTransformerId(), is(METACARD_TRANSFORMER_ID));
     }
 
     @Test
     public void testCreateRequest() throws Exception {
```

Comment:
Should add a test with 2 storage providers where first in list is mocked to fail to ensure that the other one runs.